api/server.py

```diff
@@ -217,8 +228,9 @@ def profile_model(model_id: str, document_id: str, redis=Depends(get_redis)) ->
 
 
 @app.post("/link_amr")
-def link_amr(document_id: str, model_id: str, redis=Depends(get_redis)) -> ExtractionJob:
-
+def link_amr(
+    document_id: str, model_id: str, redis=Depends(get_redis)
+) -> ExtractionJob:
```

Comment:
Function definitions should either be on a single line or have each param on a different line. Can we expand this out?